test/k8sT/KafkaPolicies.go

```diff
@@ -107,157 +106,112 @@ var _ = Describe("K8sValidatedKafkaPolicyTest", func() {
-
-		By("Waiting for all Cilium Pods and endpoints to be ready ")
-		By("Waiting for node K8s1")
-		ciliumPod1, _ := kubectl.WaitCiliumEndpointReady(podFilter, helpers.K8s1)
-
-		By("Waiting for node K8s2")
-		kubectl.WaitCiliumEndpointReady(podFilter, helpers.K8s2)
-
-		appPods := helpers.GetAppPods(apps, helpers.DefaultNamespace, kubectl, "app")
 		By("Testing basic Kafka Produce and Consume")
-
 		// We need to produce first, since consumer script waits for
 		// some messages to be already there by the producer.
-		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[empireHqApp], fmt.Sprintf(prodHqAnnounce))
-		Expect(err).Should(BeNil())
 
-		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[outpostApp], fmt.Sprintf(conOutpostAnnoune))
-		Expect(err).Should(BeNil())
+		err := kubectl.ExecKafkaPodCmd(
+			helpers.DefaultNamespace, appPods[empireHqApp], prodHqAnnounce)
+		Expect(err).Should(BeNil(), "EmpireHQApp cannot announce HQ")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[empireHqApp], fmt.Sprintf(prodHqDeathStar))
-		Expect(err).Should(BeNil())
+			helpers.DefaultNamespace, appPods[outpostApp], conOutpostAnnoune)
+		Expect(err).Should(BeNil(), "OutpostApp cannot announce Outpost")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[outpostApp], fmt.Sprintf(conOutDeathStar))
-		Expect(err).Should(BeNil())
+			helpers.DefaultNamespace, appPods[empireHqApp], prodHqDeathStar)
+		Expect(err).Should(BeNil(), "EmpireHqApp cannot produce messages")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[backupApp], fmt.Sprintf(prodBackAnnounce))
-		Expect(err).Should(BeNil())
+			helpers.DefaultNamespace, appPods[outpostApp], conOutDeathStar)
+		Expect(err).Should(BeNil(), "OutpostApp cannot produce messages")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[outpostApp], fmt.Sprintf(prodOutAnnounce))
-		Expect(err).Should(BeNil())
+			helpers.DefaultNamespace, appPods[backupApp], prodBackAnnounce)
+		Expect(err).Should(BeNil(), "BackupApp cannot not announce")
 
-		By("Apply L7 kafka policy")
-		eps1 := kubectl.CiliumEndpointPolicyVersion(ciliumPod1)
-		_, err = kubectl.CiliumPolicyAction(helpers.KubeSystemNamespace, l7Policy, helpers.KubectlApply, 300)
-		Expect(err).Should(BeNil())
-
-		By("Waiting for endpoint updates with L7 policy")
-		err = helpers.WaitUntilEndpointUpdates(ciliumPod1, eps1, 3, kubectl)
-		Expect(err).Should(BeNil())
-		epsStatus1 := helpers.WithTimeout(func() bool {
-			endpoints1, err := kubectl.CiliumEndpointsListByLabel(ciliumPod1, podFilter)
-			if err != nil {
-				return false
-			}
-			return endpoints1.AreReady()
-		}, "could not get endpoints", &helpers.TimeoutConfig{Timeout: 100})
-
-		Expect(epsStatus1).Should(BeNil())
-
-		endpoints1, err := kubectl.CiliumEndpointsListByLabel(ciliumPod1, podFilter)
-		policyStatus1 := endpoints1.GetPolicyStatus()
-
-		/*
```

Comment:
Why was this comment removed? This helps to understand the Kafka multimode setup. Could you please put it back?